packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -783,6 +852,14 @@ class IosProject extends XcodeBasedProject {
     return registryDirectory.childFile('GeneratedPluginRegistrant.m');
   }
 
+  File get lldbInitFile {
+    return ephemeralDirectory.childFile('.lldbinit');
```

Comment:
I think it might be better to not call it `.lldbinit` to avoid being hit by some zealous `.gitignore`. There is also nothing hidden about this file to use dotfile-like name.  Maybe `flutter_lldbinit`?

Reply:
Thanks for the clarification. I missed that these files are ephemeral rather than persistent. Current approach LGTM then.